htdocs/product/fournisseurs.php

```diff
@@ -933,7 +934,14 @@ function update_price_from_multicurrency() {
 
 						// Modify-Remove
 						print '<td class="center nowraponall">';
-						if ($usercancreate)
+						
+						if (!empty($conf->global->NOTMODIFYOTHERENTITYFOURNPRICE) && $productfourn->product_fourn_entity!=$conf->entity) {
+							$canmodifyotherentityfournprice = 0;
+						}else{
+							$canmodifyotherentityfournprice = 1;
+						}
```

Comment:
@mapiolca 
This verification of access rights must be done in the hook "doAction" of the Multicompany module.

Reply:
Nothing to do. Regis means that this must be done by the module (by adding a doAction hook) so not into the code of Dolibarr. So you can set canmodifyotherentityfournprice to 1 at begin of page before the action, keep the test on the var in core and let the module decide to overwrite this variable or not (this is a policy/feature to implement in module multicompany).

Reply:
Ok, @hregis. Does-it integer in Multicompany module ?